### DISCUSSION TITLE
Welcome to Visual Studio Marketplace Discussions!

### DISCUSSION THREAD

**Discussion by prashantvc** [microsoft/vsmarketplace#141] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ssbarnea** [comment]
This is indeed a welcome move. I hope this will help the team cleanup the issue tracker and start fixing the numerous bugs reported with the marketplace.

**Comment by iChenLei** [comment]
@prashantvc Please help us, extension owner can't add member as collaborator! Even member use micosoft's email product (e.g.  **@outlook.com) !
![2021-08-25 21 57 41](https://user-images.githubusercontent.com/14012511/130804281-2917feed-6958-4415-a08d-6ba36aaf3453.png)

Please don't tell us send email to vsmarketplace@microsoft.com, it's  bug and just fix it. Thanks a lot.

Releated Issue ->  https://github.com/microsoft/vsmarketplace/issues/129

We juse want add more member for better collaboratation, like npm collaborators.

![2021-08-25 22 02 01](https://user-images.githubusercontent.com/14012511/130804954-15255190-fdd0-4a7a-96cd-f864a20a813c.png)

**Reply by ssbarnea** [reply]
I can confirm that the only way to do this is to email support. It was like this for years,... bit of mindblowing.

**Reply by prashantvc** [reply]
This is something we are working on, for now please contact vsmarketplace@microsoft.com
